roles/kubernetes-apps/metallb/templates/metallb-config.yml.j2

```diff
@@ -6,6 +6,14 @@ metadata:
   name: config
 data:
   config: |
+{% if metallb_protocol == 'bgp' %}
+    peers:
+{% for peer in metallb_peers %}
```

Comment:
If metallb_protocol is bgp, is metallb_peers mandatory to be specified?
If so, it is better to implement check to verify metallb_peers is specified in the case at roles/kubernetes-apps/metallb/tasks/main.yml

Reply:
It's not immediately clear if peers are _mandatory_ for MetalLB, the documentation does say you need peer-address, peer-asn, and my-asn though, and as you noted the template modification I made makes metallb_peers a requirement.
I've added the check as requested.

Reply:
when using BGP, peers is mandatory otherwise MetalLB doesn't know what router to connect to in order to advertise the routes

> For a basic configuration featuring one BGP router and one IP address range, you need 4 pieces of information:
    - The router IP address that MetalLB should connect to,
    - The router’s AS number,
    - The AS number MetalLB should use,
    - An IP address range expressed as a CIDR prefix.

So I agree with @oomichi, we better add a validation check for `metallb_peers` if metalLB is running in BGP mode.

Reply:
A check for metallb_peers was added to roles/kubernetes-apps/metallb/tasks/main.yml in f185879c7e2ac4b108c6764a9b1bd4b39e1766fc